wordpress-mobile/WordPress-Android
# Changed thumbnail size

**Fixes:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2563

## Description
Resizes thumbnails (from 136dpx204dp to 153dpx204dp) to have the same aspect ratio with the returned thumbnails to avoid awkward cropping .

|Downloaded image|Problem|
|---|---|
|<img width="435" src="https://user-images.githubusercontent.com/304044/96848126-b3bd3380-145c-11eb-9803-9894c847a596.png">|<img width="435" src="https://user-images.githubusercontent.com/304044/96835600-79e43100-144c-11eb-8ddd-0912563fe2bc.png">|

## To test:
Layout Picker should show [when creating a new page from *My Site* or *Site Pages* the *Modal Layout Picker* appears](https://github.com/wordpress-mobile/WordPress-Android/pull/12482).

1. Fresh install the app or clear storage (Device Settings>Apps>WordPress Beta>Storage & cache>Clear storage)
1. Login
1. Open the Modal Layout Picker 
1. Verify that the thumbnails are not cropped awkwardly

## Screenshot

|Before|After|
|---|---|
|![device-2020-10-22-114307](https://user-images.githubusercontent.com/304044/96847445-dc90f900-145b-11eb-9426-0262c9fc09dc.png)|![device-2020-10-22-114238](https://user-images.githubusercontent.com/304044/96847473-e4e93400-145b-11eb-8256-ccdef1a961e5.png)|

PR submission checklist:
